netlify/next-runtime
# chore: update README image with new Next.js logo

### Summary

Removes the old version of the README image and replaces it with one donning the new Next.js logo.

### Test plan

1. Visit the [README.md](https://github.com/netlify/next-runtime/blob/kma/readme/README.md) file
2. See the change.

### Relevant links (GitHub issues, Notion docs, etc.) or a picture of cute animal
Closes https://github.com/netlify/next-runtime/issues/1593

![cute animal](https://res.cloudinary.com/cutelify/image/upload/c_fill,g_auto,q_auto:good,w_300,h_350/gary-bendig-6GMq7AGxNbE-unsplash_swy1aa.jpg)

### Standard checks:



- [ ] Check the linked README file
